src/utils/maybeParseInt.ts

```diff
@@ -0,0 +1,7 @@
+export function maybeParseInt(str: string | null): number | null {
+  if (!str) {
+    return null;
+  }
+
+  return parseInt(str);
```

Comment:
The `parseInt` function should be provided with a radix parameter to specify the base of the number in the string. Without specifying the radix, `parseInt` will default to base 10 unless the string starts with '0x' for hexadecimal or '0' for octal, which can lead to unexpected results. It's a common best practice to always include the radix to avoid such issues.